design/behavior.actions.md

```diff
@@ -0,0 +1,94 @@
+# Behavior Actions
+
+Behaviors are high level concepts that are made up of one or more actions.  These actions make up the interactions
+that the user makes with the given behavior.  These actions are broken up in to three states: `started` and `stopped`.
```

Comment:
"three"?

Reply:
Yeah.  I need to update the doc here for this.  Good catch.  The actions are broken up in to three states.